examples/llm/src/models/utils/meta_init_context.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@alextrott16 could you check with @hanlint what we have to do re. licenses here? If this gets complicated, we can rewrite the meta init function ourselves or try to use something like `torchdistx` with `deferred_init()`

Reply:
LGTM